firebase-perf/src/main/java/com/google/firebase/perf/metrics/Trace.java

```diff
@@ -507,37 +495,32 @@ protected void finalize() throws Throwable {
     }
   }
 
-  /** @hide */
   /** @hide */
```

Comment:
Do we need this? Why do we have two @hide annotations at times?

Reply:
Looks like you are looking at the older code.